### ISSUE TITLE
[Feature Proposal] "Follows You" indicator

### ISSUE THREAD

**Issue by ryanbaer** [busyorg/busy#964]
Per our new development guidelines, I wanted to have discussion / get approval to begin working on a feature I proposed on Utopian.

You can read the full post here: [Utopian: Busy Feature Idea: "Follows You" indicator](https://utopian.io/utopian-io/@ryanbaer/busy-feature-idea-follows-you-indicator)

But here are some screenshots which convey what I had in mind:
![image](https://user-images.githubusercontent.com/11914327/32244721-bd674184-be36-11e7-8d99-78f8f6a6222a.png)

![image](https://user-images.githubusercontent.com/11914327/32244747-d49d2b84-be36-11e7-865e-bf30c03f4fc6.png)

**Comment by ryanbaer** [comment]
Discussion on Discord:
- It was not preferred to show this anywhere else than in the user header - i.e. do not display it on comments or in the upvotes modal
- On both desktop and mobile, display next to the `@` handle

Please mark me as the assignee when you have a moment.

Thanks

**Comment by eastmaels** [comment]
Is this already assigned and resolved? I'd like to work on this.

**Comment by ryanbaer** [comment]
A quick note: I've had to step away from Busy due to other priorities. It was really great to work on this project with all of you - thanks for everything.

@eastmaels As for this ticket, go for it - I didn't get too far in implementation. One tip: filtering through the authenticated user's follower list for a given username can quickly become inefficient. Consider Jerry Banfield, who has nearly 18,000 followers.

Where I left off was that I had found an endpoint on the Steem API that allows you to query the user's followers: `getFollowersAsync`. I was simply querying that endpoint with the username of interest and specifying a limit of 1. The endpoint seems to always return a username, even if it isn't a match, so the result of whether or not the user is following the authenticated user is if the returned username matches the one you queried with.
If that didn't make sense, let me know, haha.

And feel free to tackle it a different way if there's a better method, that's just where I was at with it last. Excited to see it come together!
